### ISSUE TITLE
Unable to detect the tablet

### ISSUE THREAD

**Issue by stllok** [InfinityGhost/OpenTabletDriver#662]
## Description
Console just show this error
 `[ArgumentOutOfRangeException:Error]	Value range is [0, 15]. (Parameter 'value')`
![image](https://user-images.githubusercontent.com/16291220/104871163-52d2c800-5985-11eb-8252-483cb7870e7f.png)

and why the diagnostics is a blank file??

## System Information:

| Name             | Value |
| ---------------- | ----- |
| Operating System | Pop!_OS 20.10 |
| Software Version | v0.5.0 |
| Tablet           | XP-Pen G430S |

**Comment by X9VoiD** [comment]
https://github.com/InfinityGhost/OpenTabletDriver/wiki/Linux-FAQ

**Comment by stllok** [comment]
humm....
```
echo "blacklist hid-uclogic" | sudo tee -a /etc/modprobe.d/blacklist.conf
sudo rmmod hid-uclogic
```

still have a same error after run that script

![image](https://user-images.githubusercontent.com/16291220/104923341-e2ed2d80-59d6-11eb-99ad-4b3534bf152e.png)

and seem it didn't detect G430S?

btw I can't run opentabletdriver on root or sudo
```
root@pop-os:~# opentabletdriver
Failed to connect to bus: No such file or directory
Failed to connect to bus: No such file or directory

(OpenTabletDriver.UX.Gtk:8170): libnotify-WARNING **: 21:50:54.354: Failed to connect to proxy
Exception in Gtk# callback delegate
  Note: Applications can use GLib.ExceptionManager.UnhandledException to handle the exception.
System.NullReferenceException: Object reference not set to an instance of an object.
   at OpenTabletDriver.UX.MainForm.DetectAllTablets()
   at OpenTabletDriver.UX.MainForm.<ConstructMenu>b__4_7(Object sender, EventArgs e)
   at System.Threading.Tasks.Task.<>c.<ThrowAsync>b__140_0(Object state)
   at Eto.GtkSharp.Forms.GtkSynchronizationContext.<>c__DisplayClass0_0.<Post>b__0()
   at GLib.Idle.IdleProxy.Handler()
   at GLib.ExceptionManager.RaiseUnhandledException(Exception e, Boolean is_terminal)
   at GLib.Idle.IdleProxy.Handler()
   at Eto.GtkSharp.Forms.ApplicationHandler.Run()
   at Eto.Forms.Application.Run(Form mainForm)
   at OpenTabletDriver.UX.App.Run(String platform, String[] args)
   at OpenTabletDriver.UX.Gtk.Program.Main(String[] args)
```
```
stllok@pop-os:~$ sudo opentabletdriver
Failed to connect to bus: No such file or directory
Failed to connect to bus: No such file or directory

(OpenTabletDriver.UX.Gtk:8567): libnotify-WARNING **: 21:52:19.629: Failed to connect to proxy
Exception in Gtk# callback delegate
  Note: Applications can use GLib.ExceptionManager.UnhandledException to handle the exception.
System.NullReferenceException: Object reference not set to an instance of an object.
   at OpenTabletDriver.UX.MainForm.DetectAllTablets()
   at OpenTabletDriver.UX.MainForm.<ConstructMenu>b__4_7(Object sender, EventArgs e)
   at System.Threading.Tasks.Task.<>c.<ThrowAsync>b__140_0(Object state)
   at Eto.GtkSharp.Forms.GtkSynchronizationContext.<>c__DisplayClass0_0.<Post>b__0()
   at GLib.Idle.IdleProxy.Handler()
   at GLib.ExceptionManager.RaiseUnhandledException(Exception e, Boolean is_terminal)
   at GLib.Idle.IdleProxy.Handler()
   at Eto.GtkSharp.Forms.ApplicationHandler.Run()
   at Eto.Forms.Application.Run(Form mainForm)
   at OpenTabletDriver.UX.App.Run(String platform, String[] args)
   at OpenTabletDriver.UX.Gtk.Program.Main(String[] args)
```

**Comment by X9VoiD** [comment]
You're supposed to restart the daemon after running the script. And **do not** run OTD as root.

**Comment by stllok** [comment]
restarted, and still got that error

**Comment by X9VoiD** [comment]
Just the daemon lol. If you rebooted the whole system you'll have to unload the module again, or add it to blacklist so you won't have to do it every reboot. Read FAQ.

**Comment by stllok** [comment]
following the second and fifth FAQ and install the xp-pen linux driver, now it can detected. But it detecd as G540 not G430S
![image](https://user-images.githubusercontent.com/16291220/105336168-e28da600-5c13-11eb-997c-7e7dce43f6e3.png)

Weird

**Comment by X9VoiD** [comment]
@stllok Open OTD -> Tablet -> Device string reader

```String Index: 4```

Then hit `Send Request`, paste output here.

**Comment by stllok** [comment]
V2.0 20171204

**Comment by X9VoiD** [comment]
Try to get String Index from 0 to 10

**Comment by stllok** [comment]
0: Љ
1: XP-PEN
2: G430S
3-10 : V2.0 20171204

**Comment by X9VoiD** [comment]
Extract [XP-Pen.zip](https://github.com/InfinityGhost/OpenTabletDriver/files/5897371/XP-Pen.zip) to `Configurations/XP-Pen`, then press `Detect tablet` and see if that fixes the G430/S/G540 mix up.

**Comment by stllok** [comment]
![image](https://user-images.githubusercontent.com/16291220/106413801-ff34a400-6485-11eb-88ed-ffb76bc0f886.png)

G430 (sorry for the late)

**Comment by lululock71** [comment]
I just had a similar issue (Kamvas Pro 13 not detected) and running
`echo "blacklist hid-uclogic" | sudo tee -a /etc/modprobe.d/blacklist.conf
sudo rmmod hid-uclogic
`
solved my issue too. My tablet is recognized but does not work correctly (but that's for another issue haha).

**Comment by stllok** [comment]
> I just had a similar issue (Kamvas Pro 13 not detected) and running
> `echo "blacklist hid-uclogic" | sudo tee -a /etc/modprobe.d/blacklist.conf sudo rmmod hid-uclogic `
> solved my issue too. My tablet is recognized but does not work correctly (but that's for another issue haha).
> 
> I have to run it at each reboot tho, I dunno why.

humm... I can detect the tablet now, but now the problem is it did not detect correctly lol

**Comment by Mrcubix** [comment]
what do you mean by not detected correctly? also paste he new console logs here and a diagnostic file from help > export diag

**Comment by InfinityGhost** [comment]
Issue appears stale, if there's no update in 7 days it'll be closed.

**Comment by stllok** [comment]
I think it can close now since the initial problem is fixed, the only problem is #698 for now
